RELEASES.md

```diff
@@ -1,3 +1,130 @@
+  powerful functionality when combined correctly. Most notably you can
+  now include external documentation in your crate by writing the following.
+  ```rust
+  #![doc = include_str!("README.md")]
+  ```
+
+- [You can now cast between unsized slice types (and types which contain
+  unsized slices) in `const fn`.][85078]
+- [TODO: stabilize member constraints][84701]
+
+Compiler
+-----------------------
+
+- [Rustc will now search for custom JSON targets in
+  `/lib/rustlib/<target-triple>/target.json` where `/` is the "sysroot"
+  directory.][83800] You can find your sysroot directory by running
+  `rustc --print sysroot`.
+- [Added `wasm` as a `target_family` for WebAssembly platforms.][84072]
+- [You can now use `#[target_feature]` on safe functions when targeting
+  WebAssembly platforms.][84988]
+- [Improved debugger output for enums on Windows MSVC platorms.][85292]
+- [Added tier 3\* support for `bpfel-unknown-none`
+   and `bpfeb-unknown-none`.][79608]
+- [Added tier 3 support for `powerpc64le-unknown-freebsd`.][83572]
+
+Libraries
+-----------------------
+
+- [You can now explicitly import the prelude of different editions
+  through `std::prelude`. (e.g. `use std::prelude::rust_2021::*;`)][86294]
+- [`panic::panic_any` will now `#[track_caller]`.][85745]
+- [Added `OutOfMemory` as a variant of `io::ErrorKind`.][84744]
+- [ `proc_macro::Literal` now implements `FromStr`.][84717]
+- [TODO: Bump stdarch submodule][83278]
+
+Stabilized APIs
+---------------
+
+- [`BTreeMap::into_keys`]
+- [`BTreeMap::into_values`]
+- [`HashMap::into_keys`]
+- [`HashMap::into_values`]
+- [`MaybeUninit::assume_init_mut`]
+- [`MaybeUninit::assume_init_ref`]
+- [`arch::wasm32`]
+- [`arch::x86::_bittest`]
+- [`arch::x86::_bittestandcomplement`]
+- [`arch::x86::_bittestandset`]
+- [`arch::x86_64::_bittest`]
+- [`arch::x86_64::_bittestandcomplement64`]
+- [`arch::x86_64::_bittestandcomplement`]
+- [`arch::x86_64::_bittestandreset64`]
+- [`arch::x86_64::_bittestandset64`]
+- [`arch::x86_64::_bittestandset`]
+- [`ops::ControlFlow`]
+- [`VecDeque::binary_search`]
+- [`VecDeque::binary_search_by`]
+- [`VecDeque::binary_search_by_key`]
+- [`VecDeque::partition_point`]
+
+Cargo
+-----------------------
+
+- [Added the `--prune <spec>` option to `cargo-tree` to remove a package from
+  the dependency graph.][cargo/9520]
+- [Added the `--depth` option to `cargo-tree` to print only to a certain depth
+  in the tree ][cargo/9499]
+- [Added the `no-proc-macro` value to `cargo-tree --edges` to hide procedural
+  macro dependencies.][cargo/9488]
+- [A new environment variable named `CARGO_TARGET_TMPDIR` is available.][cargo/9375]
+  This variable points to a directory that integration tests and benches
+  can use as a "scratchpad" for testing filesystem operations.
+- [Configure hosts separately from targets when --target is specified.][cargo/9322]
+- [Respect Cargo.toml `[package.exclude]` even not in a git repo.][cargo/9186]
```

Comment:
This is a pretty obscure fix that is unlikely to be important to anyone except a small handful of users.  The problem was using negative rules (`!foo`) in `exclude` (thus a double-negative) when not in a git repo. Feel free to leave it in the notes, but I probably wouldn't mention it.

Reply:
Yeah both of these last two items weren't supposed to be included, I thought I removed them.